### PR TITLE
[apriltag] update to 3.4.0

### PR DESCRIPTION
**Pull request by AenBleidd**

### CODE REVIEW

File: ports/apriltag/portfile.cmake [hunk 1]
```diff
@@ -6,14 +6,17 @@ vcpkg_from_github(
...
     HEAD_REF master
     PATCHES
         fix-config.patch
 )
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DBUILD_PYTHON_WRAPPER=OFF
+        -DBUILD_EXAMPLES=OFF
```
Comment (dg0yt) [inline ports/apriltag/portfile.cmake:19]:
```suggestion
        -DBUILD_EXAMPLES=OFF
        -DBUILD_TESTING=OFF
```
and drop the patch? (If BUILD_TESTING doesn't already default to OFF).

Comment (AenBleidd) [reply ports/apriltag/portfile.cmake:19]: There is no such option as `BUILD_TESTING` in the CMakeLists.txt file

Comment (dg0yt) [reply ports/apriltag/portfile.cmake:17]: It doesn't need to be an explicit `option`. CMake users can add any variable as a `cache` variable. (However, normal variables hide cache variable.) apriltag's CI scripts set this variable to ON.

**Review by WangWeiLin-MV — APPROVED** [review]
Port usage tests pass with following triplets:
* x64-windows

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**